front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/SettingsPanel.tsx

```diff
@@ -28,12 +34,34 @@ const SettingsPanel = ({ settings, onChange, onClose }: SettingsPanelProps) => {
         <X />
       </button>
 
+      <section className="pb-3">
+        <RadioGroup
+          label={t('timeSpaceChartSettings.projection')}
+          value={settings.projectionType}
+          onChange={(value) => {
+            onChange({ ...settings, showSignalsStates: false });
+            dispatch(updateProjectionType(value as ProjectionType));
```

Comment:
To keep your commits independent, you should remove the onChange in the 3rd commit (`front: add radio button in get settings paneling`) ;)
```suggestion
            dispatch(updateProjectionType(value as ProjectionType));
```

Reply:
Done